templates/solutions/ai/index.html

```diff
@@ -296,278 +299,486 @@ <h2>Enterprise data management</h2>
+                                attrs={"class": "p-logo-section__logo"}) | safe
+                }}
+              </div>
+              <div class="p-logo-section__item">
+                {{ image(url="https://assets.ubuntu.com/v1/9c1b30d1-google-cloud-logomark%20%5BDEPRECATED%5D.png",
+                                alt="Google Cloud",
+                                width="114",
+                                height="313",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-logo-section__logo"}) | safe
+                }}
+              </div>
+              <div class="p-logo-section__item">
+                {{ image(url="https://assets.ubuntu.com/v1/08c2f850-ibm-cloud-logo.png",
+                                alt="IBM",
+                                width="437",
+                                height="313",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-logo-section__logo"}) | safe
+                }}
+              </div>
+              <div class="p-logo-section__item">
+                {{ image(url="https://assets.ubuntu.com/v1/c165433e-oracle-cloud-logo.png",
+                                alt="Oracle",
+                                width="388",
+                                height="313",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-logo-section__logo"}) | safe
+                }}
+              </div>
+            </div>
+          </div>
+          <style>
+            /* Avoid wrapping logos */
+            .p-logo-section__item {
+              margin-right: 2rem !important;
+            }
+          </style>
+        </div>
+        <hr class="p-rule--muted" />
         <p>
-          <a href="https://ubuntu.com/engage/run-ai-at-scale">Download the whitepaper on NVIDIA DGX and Charmed Kubeflow&nbsp;&rsaquo;</a>
+          <a href="https://ubuntu.com/engage/run-ai-at-scale">Discover full-stack AI infrastructure&nbsp;&rsaquo;</a>
         </p>
-        {{ image(url="https://assets.ubuntu.com/v1/ba3a0335-Canonicla+nvidia.png",
-                alt="",
-                width="536",
-                height="66",
-                hi_def=True,
-                loading="lazy") | safe
-        }}
       </div>
     </div>
   </section>
 
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+  </div>
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        <a href="https://ubuntu.com/engage/open-source-big-data-ai">Get the free guide: Build a smarter enterprise with big data and AI&nbsp;&rsaquo;</a>
+      </h2>
+    </div>
+  </section>
+
   <section class="p-section">
-    <div class="row--50-50">
+    <div class="u-fixed-width p-section--shallow">
       <hr class="p-rule" />
```

Comment:
Vanilla suggests that `u-fixed-width`/`row` should not be in same div as `p-section .*`
Could you please refractor it, by that I mean to create 2 divs above one another and put these classes in different divs.
Could you also please do this for rest of code.
Thanks